spec/recommendations-v1.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$id": "/schemas/recommendations_request",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "description": "Schema for recommendations API requests received by TorchServe",
+  "additionalProperties": false,
+  "definitions": {
+    "language": {
+      "type": "string",
+      "description": "The language code of the topic",
+      "pattern": "^[a-z]{3}(?:-[A-Z]{2})?$"
+    },
+    "topic": {
+      "type": "object",
+      "description": "A topic in the tree structure for which to generate recommendations",
+      "additionalProperties": false,
+      "properties": {
+        "id": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The ID of the topic content node on Studio"
+        },
```

Comment:
One reason that we decided to include the ID, having saw that you computed it in your example, is for logging so that we can have better insight into things from Studio into the recommendations API.

Reply:
Makes sense, yep! My only request would be not enforcing it to be a UUID, as when using this API from elsewhere we may just be generating dummy (possibly more human-readable) IDs